tests/examples/directory_input/pipelined_directory_input/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+max_batch_size_dev: null
+max_batch_size_train: 5
+max_seq_length: 1024
+number_of_dev_files: 0
+number_of_test_files: 0
+number_of_training_files: 32
+token_type_ids: true
+tokenizer_model_type: <class 'transformers.models.gpt2.configuration_gpt2.GPT2Config'>
```

Comment:
should this be a simpler string for the class name?

Reply:
Yeah good point, this is not relevant to PR though so I will keep it as a note when I work on metadata 🙇